go.mod

```diff
@@ -26,6 +26,8 @@ require (
 	golang.org/x/term v0.0.0-20210220032956-6a3ed077a48d
 	golang.org/x/time v0.0.0-20210220033141-f8bda1e9f3ba
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
+	k8s.io/api v0.21.1
+	k8s.io/apimachinery v0.21.1
 	k8s.io/client-go v0.21.1
```

Comment:
hmm, I wonder why the CLI needs k8s APIs? very strange to me

Reply:
I've updated the description and code to give more context

Reply:
Thanks! One thing I'm not sure about is how close the versions of these libraries need to be to the corresponding Kubernetes versions, because we need to make sure that the versions match the Kubernetes versions that we support.

So, will v0.21.1 - which I guess corresponds to Kubernetes 1.21.1 - work with a Kubernetes 1.13 cluster?

Reply:
no clue, but I can just lock it with the same version as the coder app